variables.tf

```diff
@@ -0,0 +1,94 @@
+variable "projectname" {
+  default = ""
```

Comment:
default = "" can break things. default = "foo" or no default to get an ERROR and then know we miss one variable in our module call.